src/EntityManager.ts

```diff
@@ -0,0 +1,17 @@
+const entities = [];
+
+const entityMap: Record<string, TEntity> = {};
+
+export type TEntity = {
+  id: string;
```

Comment:
It's worth to have a discussion about our logic structure. I.e. are we going to use my proposal for how to construct entities, if we do, this should probably be that entity type.